server/data/data-ldap/src/main/java/org/apache/james/user/ldap/ReadOnlyLDAPUser.java

```diff
@@ -129,7 +129,7 @@ public boolean verifyPassword(String password) {
         try {
             BindResult bindResult = connectionPool.bindAndRevertAuthentication(userDN.toString(), password);
             return bindResult.getResultCode() == ResultCode.SUCCESS;
-        } catch (LDAPBindException e) {
+        } catch (LDAPException e) {
```

Comment:
:-1:  I fear we reject overwise valuable information.

How about instead interception the empty password before calling the LDAP, logging the info, and then return straight away a failure?